test/distributed/_composable/fully_shard/test_fully_shard_runtime.py

```diff
@@ -46,7 +46,7 @@ class TestRuntime(FSDPTest):
 
     @property
     def world_size(self) -> int:
-        return 2
+        return torch.cuda.device_count()
```

Comment:
Why this changes? For people who use the internal server to run the tests, it may result in always using 8 GPUs to do the tests. Is that what we want?

Reply:
Would we be able to use `CUDA_VISIBLE_DEVICES=x,y python ...` internally to work around this if needed?

Reply:
We should at least use a world size of 4 because we're not testing HSDP. can we workaround internally with cuda_visible_devices as andrew mentioned?